<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/logo.png</normaloff>:/res/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>登录/字体设置</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>登录设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>服务器地址：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_hostName">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>端口：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_port">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>99999</string>
            </property>
            <property name="text">
             <string>12345</string>
            </property>
            <property name="maxLength">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>字体设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="1" column="1">
           <widget class="QLabel" name="label_font_size">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLabel" name="label_font_family">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>大小：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>字体：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="font_chooser">
            <property name="minimumSize">
             <size>
              <width>82</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	border:1px solid lightgray; background:rgb(230,230,230);
	min-Height:25px;
	min-width:80px;
}
QPushButton:hover{border-color:green; background:transparent;}</string>
            </property>
            <property name="text">
             <string>选择字体</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>147</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>配色设置</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="1">
        <widget class="QPushButton" name="notice_bg_color_4">
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border:1px solid lightgray; background:rgb(230,230,230);
	min-Height:25px;
	min-width:80px;
}
QPushButton:hover{border-color:green; background:transparent;}</string>
         </property>
         <property name="text">
          <string>还原</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="isColorTextEnabled" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="styleSheet">
          <string notr="true">QGroupBox &gt; QPushButton{border:1px solid lightgray; background:rgb(230,230,230);}
QGroupBox &gt; QPushButton:hover{border-color:green; background:transparent;}</string>
         </property>
         <property name="title">
          <string>过滤配色</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QColorButton" name="filter_fg_color">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Filter FG</string>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QColorButton" name="filter_bg_color">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Filter BG</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QColorButton" name="search_fg_color">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Search FG</string>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QColorButton" name="search_bg_color">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Search BG</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">QGroupBox &gt; QPushButton{border:1px solid lightgray; background:rgb(230,230,230);}
QGroupBox &gt; QPushButton:hover{border-color:green; background:transparent;}</string>
         </property>
         <property name="title">
          <string>日志等级配色</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="1">
           <widget class="QColorButton" name="error_bg_color">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Error  BG</string>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QColorButton" name="notice_bg_color">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Notice BG</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QColorButton" name="debug_bg_color">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Debug  BG</string>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QColorButton" name="debug_fg_color">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Debug  FG</string>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QColorButton" name="notice_fg_color">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Notice FG</string>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QColorButton" name="info_bg_color">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Info   BG</string>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QColorButton" name="info_fg_color">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Info   FG</string>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QColorButton" name="error_fg_color">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Error  FG</string>
            </property>
            <property name="isColorTextEnabled" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border:1px solid lightgray; background:rgb(230,230,230);
	min-Height:25px;
	min-width:80px;
}
QPushButton:hover{border-color:green; background:transparent;}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QColorButton</class>
   <extends>QPushButton</extends>
   <header>qcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
